updater/osv/osv.go

```diff
@@ -35,6 +35,7 @@ var Factory driver.UpdaterSetFactory = &factory{}
 type factory struct{}
 
 func (factory) UpdaterSet(context.Context) (s driver.UpdaterSet, err error) {
+	s = driver.NewUpdaterSet()
```

Comment:
for another PR, but maybe we can add a `nil` check to `Add` and default to `driver.NewUpdaterSet()` to avoid errors like this in the future

Reply:
I'd rather we solve that by making the UpdaterSet less special, but yeah something to make this less of a footgun